migration_original/ODS1Stage/tables/Show/SOLRMarket/spu_translated_SOLRMarket.sql

```diff
@@ -0,0 +1,453 @@
+CREATE OR REPLACE PROCEDURE SP_LOAD_TABLE_SOLRMarket() -- Parameters
```

Comment:
add the database and schema to the sp name
```suggestion
CREATE OR REPLACE PROCEDURE ODS1STAGE.SHOW.SP_LOAD_TABLE_SOLRMarket() -- Parameters
```